src/main/java/net/pms/configuration/ConfigurableProgramPaths.java

```diff
@@ -16,27 +16,31 @@
  */
 package net.pms.configuration;
 
-import net.pms.util.ProgramExecutableType;
 import static org.apache.commons.lang3.StringUtils.isBlank;
+
 import java.nio.file.Files;
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
+
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.ThreadSafe;
-import net.pms.util.ExternalProgramInfo;
-import net.pms.util.ExecutableInfo;
-import net.pms.util.FFmpegProgramInfo;
-import net.pms.platform.PlatformProgramPaths;
-import net.pms.util.FileUtil;
+
```

Comment:
There are a few files here with these blank lines added in the imports, can that be reverted?

Reply:
Done